src/game.cpp

```diff
@@ -8542,12 +8542,12 @@ void game::zones_manager()
                                            pointPlayer.x,
                                            pointPlayer.y
                                           ),
-                              direction_name_short(direction_from(pCenter.x,
-                                                   pCenter.y,
-                                                   pointPlayer.x,
-                                                   pointPlayer.y
-                                                                 )
-                                                  ).c_str()
+                              direction_name(direction_from(pCenter.x,
```

Comment:
The suffix `_short` seems much better than a boolean parameter - which will most likely always be a compile time constant anyway.

It's far easer to read and understand what `direction_name_short(...)` means than what `direction_name(..., true)` means. What is that true? Is the parameter `long_name` or is it `short_name` or is it `untranslated` or `only_2d` or ...

One has to look up the function declaration / maybe even the implementation. As that parameter (as said above) is compile time const anyway, you don't the variability of a parameter.